bitnami/external-dns/Chart.yaml

```
@@ -28,4 +28,4 @@ maintainers:
 name: external-dns
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/external-dns
-version: 8.0.2
+version: 8.0.3
```

```suggestion
version: 8.2.0
```

Let's bump the version as minor, as this PR is introducing new parameters in `values.yaml`

---

I made the change while resolving merge conflicts